src/Locale/helpers.js

```diff
@@ -0,0 +1,45 @@
+import Utils from '../Utils/index'
+
+export const loadTranslationFile = (isoLocale, defaultLocale) => {
+  const localePath = Utils.asset(`locale/${isoLocale}.json`)
+  const localeDefaultPath = Utils.asset(`locale/${defaultLocale}.json`)
+
+  return fetch(localePath).then(response => {
+    if (response.ok) return response.json()
+    console.warn(`Locale: ${isoLocale}.json - ${response.statusText}`)
+    console.warn(`Locale: Defaulting to /${defaultLocale}.json`)
```

Comment:
Better to use Log plugin so it can be turned off easily in production

Reply:
Changed to `Log` class